src/core/lib/transition-manager.js

```diff
@@ -2,6 +2,7 @@
 import LinearInterpolator from '../transitions/linear-interpolator';
 import {extractViewportFrom} from '../transitions/transition-utils';
 import assert from 'assert';
+import PropTypes from 'prop-types';
```

Comment:
Nit: Using react's prop-types (even though they are broken our) in non-react apps is a little confusing. Especially as we are drawing a line between the react and non-react parts. Fine for now but something to consider.